### PR TITLE
Game stats corrected game update

### PR DESCRIPTION
**Pull request by jbreit88**
Added 15 passing tests to check initialized variables and methods in Game class.

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        away_team_id: "1",
+        home_team_id: "2",
+        away_goals:   "7",
+        home_goals:   "7",
+        venue:        "Toyota Stadium",
+        venue_link:   "/api/v1/venues/null"
+        })}
+
+  describe '#initialize' do
+    it 'exits' do
```
Comment (ParkerLockhart) [inline spec/game_spec.rb]: Just a little typo right here on 'exists'

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -142,6 +143,18 @@
     it 'returns hash with sesaon id as key and float of average number of goals per game as value' do
```
Comment (ParkerLockhart) [inline spec/stat_tracker_spec.rb]: Typo here in "season"

**Review by ParkerLockhart — CHANGES_REQUESTED** [review]
Overall looks amazing! I love the methods for this game class, and your tests for the attributes and methods are really thorough.

File: spec/game_spec.rb (hunk 1 above)
Comment (jbreit88) [reply spec/game_spec.rb]: Fixed. Should be good to go. Nice eyes! See commits below.

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (jbreit88) [reply spec/stat_tracker_spec.rb]: Fixed. :) See commits below

**Review by joel-grant — APPROVED** [review]

**Review by ParkerLockhart — APPROVED** [review]
Awesome!!

**Pull request merged**